koku/currency/common.py

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2021 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Common utilities and helpers for Currency."""
+import json
+import os
+
+from tenant_schemas.utils import schema_context
+
+from koku.settings import KOKU_DEFAULT_CURRENCY
+from reporting.currency.models import CurrencySettings
+
+CURRENCY_FILE_NAME = f"{os.path.dirname(os.path.realpath(__file__))}/specs/currencies.json"
+
+
+def load_currencies_from_file(file_path=CURRENCY_FILE_NAME):
+    with open(file_path) as api_file:
+        data = json.load(api_file)
+        return data
+
+
+def get_selected_currency_or_setup(schema):
```

Comment:
Maybe we can move this to a new middleware or to the koku user middelware to set the currency setting to the KOKU_DEFAULT_CURRENCY on account creation?